tox.ini

```diff
@@ -9,7 +10,18 @@ deps =
 commands =
 	py.test {posargs}
 
+[testenv:negtz]
+passenv = LANG
+setenv =
+    TZ = America/Argentina/Buenos_Aires
+deps =
```

Comment:
I mean this part:

    Defaults for each setting in this section are looked up in the:

    [testenv]
    ...

    testenvironment default section.